CHANGELOG.md

```diff
@@ -3,10 +3,9 @@
 <!--## Unreleased-->
 ## v2.10.1
 
-- Provide a way to update application process by a known process name
+- Add GetChildProcessById, GetAllProcessByName and some methods to fix issue close/quit application

```

Comment:
@quangnt this is not addressed as suggested. We need to general this, not too narrow `to fix issue close/quit application` - because the new functions can be used for general purposes.